app/helpers/format_helper.rb

```diff
@@ -102,13 +102,12 @@ def class_for_todo(bool)
     end
   end
 
-  # rubocop:disable Lint/EndAlignment
```

Comment:
Unrelated to this PR, at least move this to a different commit which explain what you are doing here